samples/CommunityToolkit.Maui.Sample/Pages/Alerts/SnackbarPage.xaml.cs

```diff
@@ -75,13 +76,22 @@ async void DisplayCustomSnackbarButtonClicked(object? sender, EventArgs args)
 				customSnackbar.Dispose();
 			}
 
-			DisplayCustomSnackbarButton.Text = displayCustomSnackbarText;
+			DisplayCustomSnackbarButtonAnchoredToButton.Text = DisplayCustomSnackbarText;
 		}
 		else
 		{
-			throw new NotSupportedException($"{nameof(DisplayCustomSnackbarButton)}.{nameof(ITextButton.Text)} Not Recognized");
+			throw new NotSupportedException($"{nameof(DisplayCustomSnackbarButtonAnchoredToButton)}.{nameof(ITextButton.Text)} Not Recognized");
 		}
 	}
+	
+	async void DisplaySnackbarButtonAnchoredToButtonOnBottomClicked(object sender, EventArgs e)
+	{
+		await DisplaySnackbarButtonAnchoredToButtonOnBottom.DisplaySnackbar(
+			"This is Snackbar is anchored to the button on the bottom of a page. This Snackbar appears above the button to avoid clipping the Snackbar on the bottom of the Page.",
```

Comment:
```suggestion
			"This Snackbar is anchored to the button on the bottom of a page. This Snackbar appears above the button to avoid clipping the Snackbar on the bottom of the Page.",
```